src/Tests/Storage.fs

```diff
@@ -183,4 +183,21 @@ let tests = testList "Storage Tests" [
         Expect.equal resource.Kind "StorageV2" "Kind"
         Expect.equal resource.Sku.Name "Premium_LRS" "Sku Name"
     }
+    test "Sets blob access tier correctly different SKU kinds correctly" {
+        let account = storageAccount { default_blob_access_tier Cool }
+        let resource = arm { add_resource account } |> getStorageResource
+        Expect.equal resource.AccessTier (Nullable AccessTier.Cool) "Access Tier"
+
+        let account = storageAccount { default_blob_access_tier Hot }
+        let resource = arm { add_resource account } |> getStorageResource
+        Expect.equal resource.AccessTier (Nullable AccessTier.Cool) "Access Tier"
```

Comment:
Shouldn't this be Hot?

Reply:
Yes. Don't know how that slipped through :(